### PR TITLE
added changes suggested by frenk in previous merged documentation

### PR DESCRIPTION
**Pull request by pjanuario**
## Proposed change
This PR is a follow up to integrate changes requested at https://github.com/home-assistant/home-assistant.io/pull/28953

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ecoforest.markdown [hunk 1]
```diff
@@ -14,28 +14,14 @@ ha_platforms:
...
-</div>
-
-### Credentials
-
-To configure the Ecoforest integration you will need to enter your Ecoforest credentials which are the same ones you would use with the manufacturer app. The image below shows how to obtain the credentials:
-
-- Username: use the serial number of the device as identified by 1.
-- Password: use the first 8 characters of the wifi password as identified by 4.
-
-![Ecoforest Credentials](/images/integrations/ecoforest/credentials.png)
```
Comment (frenck) [inline source/_integrations/ecoforest.markdown]: Why has this been removed?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ecoforest.markdown (hunk 1 above)
Comment (pjanuario) [reply source/_integrations/ecoforest.markdown]: https://github.com/home-assistant/home-assistant.io/pull/28953#discussion_r1333168915
> @joostlek @pjanuario It doesn't make much sense to have install instructions above how to get the details needed to install. This is a prerequisite...

This was my understanding from previous comment in the other PR, maybe I misunderstood. So what do you mean exactly?

Comment (frenck) [reply source/_integrations/ecoforest.markdown]: I mean this should have been listed before/above the config flow procedure (and the paragraph to be renamed to "Prerequisites" to match our common standards).

../Frenck

**Review by frenck — APPROVED** [review]
Thanks, @pjanuario 👍 

../Frenck

**Pull request merged**